src/SupervisionSuccess/constants.js

```diff
@@ -1,2 +1,4 @@
-/* eslint-disable import/prefer-default-export */
-export const CSV_PROCESSING_ERROR = "Error occurred while processing csv file";
+export const ERROR_RESPONSE_NOT_OK = "Could not load data file (response not OK)";
+export const ERROR_NOT_CSV_FETCHED =
+  "Could not load data file (fetched not csv file). Most likely the 404 Page was received.";
```

Comment:
Let's change this to: "Could not load the params file, either because the file was not found or it was not of type 'text/csv'"